raiden_contracts/tests/test_channel_open.py

```diff
@@ -61,26 +61,29 @@ def test_open_channel_state(token_network, get_accounts):
     (A, B) = get_accounts(2)
     settle_timeout = SETTLE_TIMEOUT_MIN + 10
 
+    channel = token_network.call().getChannelInfo(1)
+    assert channel[0] == 0  # settle_block_number
+    assert channel[1] == 0  # state
+
     token_network.transact().openChannel(A, B, settle_timeout)
 
     channel = token_network.call().getChannelInfo(1)
-    assert channel[0] == settle_timeout
-    assert channel[1] == empty_address
-    assert channel[2] == 0
-
-    A_state = token_network.call().getChannelParticipantInfo(1, A)
-    assert A_state[0] is True
-    assert A_state[1] == 0
-    assert A_state[2] == 0
-    assert A_state[3] == 0
-    assert A_state[4] == fake_bytes(32)
-
-    B_state = token_network.call().getChannelParticipantInfo(1, B)
-    assert B_state[0] is True
-    assert B_state[1] == 0
-    assert B_state[2] == 0
-    assert B_state[3] == 0
-    assert B_state[4] == fake_bytes(32)
+    assert channel[0] == settle_timeout  # settle_block_number
+    assert channel[1] == 1  # state
+
+    A_state = token_network.call().getChannelParticipantInfo(1, A, B)
+    assert A_state[0] == 0
```

Comment:
For this pattern you use here in the tests I would like some more readability.

It's very hard to see what you are testing here for. Can you turn the `ParticipantInfo` into a named tuple? Perhaps by wrapping the call to `getChannelParticipantInfo()` for the tests and make sure you always return a named tuple.

This way all your tests below will look like this:

```python
assert A_state.initialized is True
assert A_state.is_closer is False
assert A_state.balance_hash_or_locksroot == fake_bytes(32)
assert A_state.nonce_or_locked_Amount == 0
```

And generally all your tests in all of the files should change to use the above pattern.
Can be done in a later PR too, just please don't forget this.

Reply:
done in https://github.com/raiden-network/raiden-contracts/pull/39/commits/a6fd4b5c11e52a1328331474fcc9a4c01cde13dc